addons/craft/create_recipes.py

```diff
@@ -204,10 +208,10 @@ def build_recipe_string(name, crystal, ingredients):
 
 def save_recipes(recipes):
     with open('recipes.lua', 'w') as fd:
```

Comment:
Would this (and other) `open` functions benefit from the `encoding` parameter? You added to to the file you're reading from, but I'm not sure what exactly we're writing here. Can non-ASCII characters appear, like Japanese text?